app/templates/public/index.hbs

```diff
@@ -8,11 +8,11 @@
   {{#if (and this.model.event.isOneclickSignupEnabled this.model.tickets)}}
     <h2>{{t 'Registration'}}</h2>
     {{!-- template-lint-disable style-concatenation --}}
-    {{#each (sort-by 'position' this.model.tickets) as |ticket| }}
-    <div role='button' onclick={{action "selectTicket" ticket}} class="ui segment {{if (eq this.selectedRegistration ticket) 'select-ticket'}}" style="cursor:pointer; max-width:{{if device.isMobile '100%' '50%'}}">
-        {{!-- template-lint-disable no-invalid-interactive --}}
-        {{#if (includes ticket.type 'Registration')}}
-          <div class="ui two column grid"  style="align-items: center;">
+     {{#each (sort-by 'position' this.model.tickets) as |ticket| }}
+       {{#if (includes ticket.type 'Registration')}}
+         <div role='button' onclick={{action "selectTicket" ticket}} class="ui segment {{if (eq this.selectedRegistration ticket) 'select-ticket'}}" style="cursor:pointer; max-width:{{if device.isMobile '100%' '50%'}}">
+            {{!-- template-lint-disable no-invalid-interactive --}}
+           <div class="ui two column grid"  style="align-items: center;">
```

Comment:
fix indentation

Reply:
Indentation is already fixed. I have done that already